packages/grid/_modules_/grid/components/menu/columnMenu/GridColumnMenu.tsx

```diff
@@ -19,26 +20,27 @@ export interface GridColumnMenuProps extends React.HTMLAttributes<HTMLUListEleme
 export const GridColumnMenu = React.forwardRef<HTMLUListElement, GridColumnMenuProps>(
   function GridColumnMenu(props: GridColumnMenuProps, ref) {
     const { hideMenu, currentColumn, open, id, labelledby, className, ...other } = props;
+    const strippedProps = useStrippedProps(other);
     const handleListKeyDown = React.useCallback(
       (event: React.KeyboardEvent) => {
-        if (event.key === 'Tab') {
+        if (isTabKey(event.key)) {
```

Comment:
I already done that. Check https://github.com/mui-org/material-ui-x/blob/master/packages/grid/_modules_/grid/components/menu/columnMenu/GridColumnMenu.tsx#L22
You need to rebase

Reply:
That is true. The reason why git shows this as diff is that the indentation of the entire block is different.